desktop/lib.go

```diff
@@ -61,20 +54,6 @@ var issueMap = map[string]string{
 	"Other":                       "9",
 }
 
-func init() {
-	cdir := configDir(&flags)
-	ss := settings.LoadSettings(cdir)
-	userConfig := func() common.UserConfig {
-		return settings.UserConfig(ss)
-	}
-	proClient = proclient.NewClient(fmt.Sprintf("https://%s", common.ProAPIHost), &webclient.Opts{
-		UserConfig: userConfig,
-	})
-	authClient = auth.NewClient(fmt.Sprintf("https://%s", common.DFBaseUrl), userConfig)
-
-	a = app.NewApp(flags, cdir, proClient, ss)
```

Comment:
I'm moving the initialization code here to Run() to have more control over when and how it is invoked and to avoid relaying on globals